build/Build.Pack.cs

```diff
@@ -538,15 +538,15 @@ string ConstructRedHatPackageFilename(string packageName, string architecture)
             const string author = "Octopus Deploy";
             const string title = "Octopus Tentacle cross platform bundle";
 
-            OctopusTasks.OctopusPack(o => o
```

Comment:
The Octo CLI used by `OctopusTasks` is the old Octopus CLI, which requires .NET6 and is no longer maintained.

Consequently, this has been updated to use the new CLI.